Swiftification/ObserverSet.swift

```diff
@@ -28,41 +28,42 @@
 
 import Dispatch
 
-public class ObserverSetEntry<Parameters> {
+open class ObserverSetEntry<Parameters> {
```

Comment:
This is what the converter did.  I'm not sure what `open` does in Swift 3.  If it performs the same function as `public` wouldn't we want to keep it though?